locales/data.json

```diff
@@ -81,16 +81,8 @@
     "GCPDesc": "DE Raw cost from Google Cloud Platform infrastructure.",
     "GroupByAll": "DE All {groupBy}s",
     "GroupByTop": "DE Top {groupBy}s",
-    "GroupByTopValueAccount": "DE accounts",
-    "GroupByTopValueCluster": "DE clusters",
-    "GroupByTopValueInstanceType": "DE Instance types",
-    "GroupByTopValueNode": "DE Nodes",
-    "GroupByTopValueOrganizationalUnitID": "Organizational units",
-    "GroupByTopValuePod": "DE Pods",
-    "GroupByTopValueProject": "DE Projects",
-    "GroupByTopValueRegion": "DE Regions",
-    "GroupByTopValueService": "DE Services",
```

Comment:
Were these keys unused? 

Can we please make GroupByTop look similar to GroupByValues? The group_by key is not localized.

<img width="533" alt="Screen Shot 2021-08-19 at 10 27 43 AM" src="https://user-images.githubusercontent.com/17481322/130086656-ab1f7173-5c17-4c50-a0e2-e8516fbbb433.png">